release/release_tests.yaml

```diff
@@ -614,7 +614,7 @@
     cluster_compute: compute_cpu_16.yaml
```

Comment:
We'll need to change `cluster_env` right? This PR adds `release/air_tests/air_benchmarks/mlperf-train/app_config_oom.yaml`, but we need to delete the old one and use the new one here